cdk/ecs/lib/stacks/ecsStack.ts

```diff
@@ -0,0 +1,754 @@
+                subnetType: SubnetType.PRIVATE_WITH_EGRESS,
+            },
+            assignPublicIp: false,
+            desiredCount: 1,
+        });
+    }
+
+    replaceRemoteServicesNames() {
+        this.DISCOVERY_SERVER_CW_CONFIG = {
+            selectors: [
+                {
+                    dimension: 'RemoteService',
+                    match: `${this.DISCOVERY_SERVER}-DNS.${this.serviceDiscoveryStack.namespace.namespaceName}*`,
+                },
+            ],
+            replacements: [
+                {
+                    target_dimension: 'RemoteService',
+                    value: `${this.DISCOVERY_SERVER}`,
+                },
+            ],
+            action: 'replace',
+        };
+
+        this.CONFIG_SERVER_CW_CONFIG = {
+            selectors: [
+                {
+                    dimension: 'RemoteService',
+                    match: `${this.CONFIG_SERVER}-DNS.${this.serviceDiscoveryStack.namespace.namespaceName}*`,
+                },
+            ],
+            replacements: [
+                {
+                    target_dimension: 'RemoteService',
+                    value: `${this.CONFIG_SERVER}`,
+                },
+            ],
+            action: 'replace',
+        };
+
+        this.VETS_SERVICE_CW_CONFIG = {
+            selectors: [
+                {
+                    dimension: 'RemoteService',
+                    match: `${this.VETS_SERVICE}-DNS.${this.serviceDiscoveryStack.namespace.namespaceName}*`,
+                },
+            ],
+            replacements: [
+                {
+                    target_dimension: 'RemoteService',
+                    value: `${this.VETS_SERVICE}`,
+                },
+            ],
+            action: 'replace',
+        };
+
+        this.VISITS_SERVICE_CW_CONFIG = {
+            selectors: [
+                {
+                    dimension: 'RemoteService',
+                    match: `${this.VISITS_SERVICE}-DNS.${this.serviceDiscoveryStack.namespace.namespaceName}*`,
+                },
+            ],
+            replacements: [
+                {
+                    target_dimension: 'RemoteService',
+                    value: `${this.VISITS_SERVICE}`,
+                },
+            ],
+            action: 'replace',
+        };
+
+        this.CUSTOMERS_SERVICE_CW_CONFIG = {
+            selectors: [
+                {
+                    dimension: 'RemoteService',
+                    match: `${this.CUSTOMERS_SERVICE}-DNS.${this.serviceDiscoveryStack.namespace.namespaceName}*`,
+                },
+            ],
+            replacements: [
+                {
+                    target_dimension: 'RemoteService',
+                    value: `${this.CUSTOMERS_SERVICE}`,
+                },
+            ],
+            action: 'replace',
+        };
+    }
+
+    createJavaTaskDefinition(serviceName: string, config: string) {
+        const { image, environmentArgs, port, rules } = JSON.parse(config);
```

Comment:
This `JSON.parse` doesn't seem to necessary. Can you just pass the json object directly? 

```
const visitsConfig: string = JSON.stringify({
            image: 'spring-petclinic-visits-service',
            environmentArgs: {
                REGION_FROM_ECS: this.region,
                CONFIG_SERVER_URL: `http://${this.CONFIG_SERVER}-DNS.${this.serviceDiscoveryStack.namespace.namespaceName}:8888`,
                DISCOVERY_SERVER_URL: `http://${this.DISCOVERY_SERVER}-DNS.${this.serviceDiscoveryStack.namespace.namespaceName}:8761/eureka`,
                VISITS_SERVICE_IP: `${this.VISITS_SERVICE}-DNS.${this.serviceDiscoveryStack.namespace.namespaceName}`,
            },
            port: 8082,
            rules: [this.DISCOVERY_SERVER_CW_CONFIG, this.CONFIG_SERVER_CW_CONFIG],
        });

        const taskDefinition = this.createJavaTaskDefinition(this.VISITS_SERVICE, visitsConfig);
```